### PR TITLE
#99_Crud_for_transport

### PR DESCRIPTION
**Pull request by swem-st**

### CODE REVIEW

File: src/app/components/home-page/home-page.component.html [hunk 1]
```diff
@@ -5,4 +5,4 @@
   <ng-template #suffixButton>
     <button nz-button nzType="primary" nzSize="large" nzSearch>Search</button>
   </ng-template>
-  <div>
\ No newline at end of file
+</div>
```
Comment (ICHTLAY) [inline src/app/components/home-page/home-page.component.html]: Add empty line at end of file

File: src/app/components/layout/layout.component.css [hunk 2]
```diff
@@ -44,3 +44,14 @@ nz-footer {
...
+    margin-right: 8px;
+    margin-bottom: 8px;
+  }
+  .transp {     
+    background-color: Transparent;
+    background-repeat:no-repeat;
+    border: none;
+    cursor:pointer;
+    overflow: hidden;        
+}
```
Comment (ICHTLAY) [inline src/app/components/layout/layout.component.css:57]: Add empty line at end of file

File: src/app/components/layout/layout.component.html [hunk 3]
```diff
@@ -4,12 +4,40 @@
...
-            <li nz-menu-item> <a *ngIf="this.accountService.isLoggedIn" (click)="logout()">Logout</a></li>
-            <li nz-menu-item> <a *ngIf="this.accountService.isLoggedIn" routerLink="/personal">Personal cabinet</a></li>
-            <li nz-menu-item> <a *ngIf="this.accountService.userRole ==='admin'">Admin Panel</a></li>
+            <li nz-menu-item>
+                <button class="transp"  nz-menu-item    
+                routerLink="/home">Home</button>
+             </li>
+            <li nz-menu-item>
+                <button class="transp"  nz-menu-item    
+                *ngIf="!this.accountService.isLoggedIn" routerLink="/signin">Sign Ip</button>
```
Comment (ICHTLAY) [inline src/app/components/layout/layout.component.html]: Change 'Sign Ip' to 'Sign In' (or maybe to 'Log in')

File: src/app/components/login-form/login-form.component.ts [hunk 4]
```diff
@@ -2,7 +2,6 @@ import { Component, OnInit } from '@angular/core';
 import { AccountService } from 'src/app/services/account.service';
 import { Router } from '@angular/router';
 import { FormBuilder, FormGroup } from "@angular/forms";
-
```
Comment (ICHTLAY) [inline src/app/components/login-form/login-form.component.ts]: Add empty line after imports

File: src/app/components/transport/transport.component.css [hunk 5]
```diff
@@ -1,52 +1,10 @@
...
-
-.notification{
-    margin: 15px auto;
-    text-align: center;
-    width: 300px;
-    height: 50px;
-}
+  nz-button-group [nz-button] {
+    margin-right: 0;
+  }
```
Comment (ICHTLAY) [inline src/app/components/transport/transport.component.css]: Add empty line at end of file

File: src/app/components/transport/transport.component.html [hunk 6]
```diff
@@ -1,66 +1,144 @@
...
+                </div>
+            </div>
+            <div nz-row>
+                <button (click)="handleOk()" nz-button [nzSize]="size" nzShape="round" [nzType]="'primary'"
+                    type="submit" [disabled]="transportForm.invalid">SUBMIT</button>
+            </div>
+        </form>
+    </ng-template>
+    <ng-template #modalFooter></ng-template>
+</nz-modal>
```
Comment (ICHTLAY) [inline src/app/components/transport/transport.component.html]: Add empty line at end of file

File: src/app/components/transport/transport.ts [hunk 7]
```diff
@@ -1,51 +1,119 @@
...
+      this.isVisible = false;
+      this.isConfirmLoading = false;
+    }, 1000);
+  }
+  handleCancel(): void {
+    console.log('Button cancel clicked!');
+    this.isVisible = false;
   }
-}
+}
```
Comment (ICHTLAY) [inline src/app/components/transport/transport.ts]: Add empty line at end of file

File: src/styles.css [hunk 8]
```diff
@@ -1,3 +1,30 @@
...
+  
+  table tbody tr:hover {
+    background-color: #fffbf2;
+    cursor: pointer;
+  }
+  
+  /*for invalid form controls*/
+  input.invalid {
+    border-color: red;
+  }
```
Comment (ICHTLAY) [inline src/styles.css]: Add empty line at end of file

File: src/app/components/transport/transport.ts [hunk 9]
```diff
@@ -1,51 +1,119 @@
...
+  onDelete(id:string){
+    if(confirm("Are you sure to delete this Transport"))
+    {
+    this.transportService.deleteTransport(id)
+    .subscribe(
+      res=>{
+        this.transportService.refreshList();
+        this.toastr.error("Car Deleted ", "congratulation")
+      },
+      err=>{console.log(err);}
```
Comment (ICHTLAY) [inline src/app/components/transport/transport.ts:110]: Only console.log in handling errors?

**Review by vladtymo — APPROVED** [review]

**Review by ICHTLAY — APPROVED** [review]

**Pull request merged**